statsforecast/utils.py

```diff
@@ -292,6 +292,10 @@ def _ensure_float(x: np.ndarray) -> np.ndarray:
         x = x.astype(np.float32)
     return x
 
+
+def _is_constant(x):
+    return np.all(x[0] == x)
```

Comment:
should we use 

https://github.com/Nixtla/statsforecast/blob/23440007294bc2d08c3753a0c84da1c219f0b0e1/statsforecast/arima.py#L1576-L1577 ?

Reply:
Changed in d222d6c

Reply:
💙